src/citrine/seeding/dataset_deletes.py

```diff
@@ -0,0 +1,53 @@
+"""Functions that perform dataset deletes."""
+from typing import List, Tuple
+
+from citrine.resources.api_error import ApiError
+from citrine.resources.dataset import Dataset
+
+from gemd.entity.link_by_uid import LinkByUID
+
+
+def clear_dataset(dataset: Dataset, delete_templates: bool = False) -> \
```

Comment:
I always find naming hard, but would 'wipe' or 'clean' be a better word here?

Reply:
Oh yes! I like those names much better. Out of the two, I prefer `wipe_dataset`. How about yourself?
@kroenlein does that sound ok with you?

Reply:
Both suggestions work.  `clean` sounds a bit more professional, but `wipe` is less likely to be confusing (I think).

Reply:
You know our customers, they get confused a lot 😅 I'll go with wipe, then 👍